### PR TITLE
Integrate remote builds

### LINKED ISSUE #257: Enable remote docker builds
**Is your feature request related to a problem? Please describe.**
I want to build on remote

### LINKED ISSUE #355: Inject docker,kubectl,okteto... binaries if namespace is managed by okteto
Also, set the `DOCKER_HOST` envvar to allow remote builds.

### PR DESCRIPTION
**Pull request by pchico83**
Closes #355
Closes #257

### CODE REVIEW

File: cmd/up.go [hunk 1]
```diff
@@ -443,6 +443,8 @@ func (up *UpContext) runCommand() error {
 	}
 
 	log.Infof("starting remote command")
+	command := strings.Join(up.Dev.Command, "\" \"")
```
Comment (rberrelleza) [inline cmd/up.go]: could we move this to a separate function and add tests?

File: cmd/up.go [hunk 2]
```diff
@@ -443,6 +443,8 @@ func (up *UpContext) runCommand() error {
...
+	command = fmt.Sprintf("PATH=$PATH:/var/okteto/bin \"%s\"", command)
```
Comment (rberrelleza) [inline cmd/up.go]: I wonder if this is somethign we should instead do in the dev env container image

Comment (pchico83) [reply cmd/up.go]: All the point is to make them available and transparently in every dev image.
I have tried another approach that I like it better.
We run a custom exec command before the user start command to kill previouos command executions. We can add logic there to move the binaries from `/var/okteto/bin` to `/usr/local/bin` if they don't exist in `/usr/local/bin` (`cp -n`).

**Comment by pchico83** [comment]
@rberrelleza merging. `OKTETO_CONTINUOUS_DEVELOPMENT` works as feature flag for remote builds too

**Pull request merged**